spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require 'simplecov'
```

Comment:
I would put this is rails_helper instead as best to keep spec_helper slim.

also consider adding config to exclude files from coverage 
[example config](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/spec/rails_helper.rb#L15-L42)

Reply:
👍